### PR TITLE
No server

### PR DESCRIPTION
**Pull request by titusfortner**
I couldn't get the server working on Windows, so I had to modify the specs to run without a server. Most of the ones marked as requiring server, do not actually, and this gives a toggle we can set to run tests without the server.

### CODE REVIEW

**Comment by p0deje** [comment]
I'm not sure I like this solution for a couple of reasons:

1. `not_compliant_on` states that current watir implementation cannot satisfy a particular part of watirspec. If we're using it to just disable a part of specs (e.g. `:no_server`) due to runtime limitations, that makes this guard pointless. From this point of view, it makes sense to introduce another guard like `requires_server(&blk)`.
2. However, it's definitely possible to make server work correctly on windows, so I'd say we should spend more time trying to make it work rather than adding yet another guard.

And :+1: for cleaning specs that don't really require server.

**Comment by titusfortner** [comment]
I'm all for getting the server up and running on Windows, but that isn't something I'd be good at doing.

I figured that creating a special guard for no_server was more descriptive than just blocking [:webdriver, :windows]. I'm completely fine with it being temporary, but I can't run any tests without it, and rebasing for everything is killing me. Can I pull them in with a FIXME tag to remove once we get the windows server running?

**Comment by p0deje** [comment]
Care to give me some time to try and make server running for you?

**Comment by titusfortner** [comment]
I updated this to just be removing no_server from places it isn't needed, and moving the watir-webdriver files into the watirspec project to accommodate the file handling method.

File: browser_spec.rb [hunk 1]
```diff
@@ -167,12 +167,11 @@
   end
 
   describe "#goto" do
-    not_compliant_on %i(webdriver internet_explorer) do
+    not_compliant_on %i(webdriver internet_explorer), :no_server do
```
Comment (p0deje) [inline browser_spec.rb]: Guess you missed this one.

**Comment by p0deje** [comment]
:+1:

**Comment by p0deje** [comment]
I've pushed https://github.com/watir/watirspec/commit/6c194728461fdbd9c7a99f44a841f939c34d0352 which fixed the problem of server start on Windows 7 for me. @titusfortner Can you try and see if it works for you?

**Pull request merged**